src/lib.rs

```diff
@@ -62,6 +62,10 @@
 //!       “enum: variant”. When added to an enum, the doc comment on the enum
 //!       becomes mandatory. When added to any other type, it has no effect.
 //!
+//! - In case you want to have an independent doc comment, the
+//!   `#[displaydoc("...")` atrribute may be used on the variant or struct to
+//!   override it.
+//!
```

Comment:
Can you also add this to the `README`, I'm pretty sure I have displaydoc setup so that the readme and the lib.rs root crate docs are supposed to be identical. If they're not then ignore this and I can fix that later.